File: src/test/scala/cern/colt/matrix/tdouble/impl/DenseLargeDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.DenseLargeDoubleMatrix2D

class DenseLargeDoubleMatrix2DViewTest(arg0: String) extends DenseLargeMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new DenseLargeDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new DenseLargeDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new DenseLargeDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}

File: src/test/java/cernx/colt/matrix/tdouble/impl/DenseLargeDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class DenseLargeDoubleMatrix2DViewTest extends DenseLargeDoubleMatrix2DTest {

    public DenseLargeDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseLargeDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseLargeDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseLargeDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}
